data/sql/updates/pending_db_world/rev_1553973207096786300.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1553973207096786300');
+
+DELETE FROM `gameobject_loot_template` WHERE `Entry`=18092 AND `Item`=2835;
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (18092, 2835, 0, 88, 0, 1, 0, 1, 6, NULL);
+
+DELETE FROM `gameobject_loot_template` WHERE `Entry`=1502 AND `Item`=2835;
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (1502, 2835, 0, 80, 0, 1, 0, 1, 11, NULL);
+
+DELETE FROM `gameobject_loot_template` WHERE `Entry`=1735 AND `Item`=2835;
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (1735, 2835, 0, 80, 0, 1, 0, 1, 6, NULL);
+
+DELETE FROM `gameobject_loot_template` WHERE `Entry`=2626 AND `Item`=2835;
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES (2626, 2835, 0, 80, 0, 1, 0, 1, 6, NULL);
```

Comment:
multiple `DELETE` or `INSERT` statements referring to the same table should be combined into the same statement, look at:

http://www.azerothcore.org/wiki/Dealing-with-SQL-files#write-compact-code

for example in your case the DELETE would be:

```sql
DELETE FROM `gameobject_loot_template` WHERE `Item`=2835 AND `Entry` IN (....);
```